install_config/install/stand_alone_registry.adoc

```diff
@@ -11,14 +11,30 @@
 
 toc::[]
 
-== Overview
+[[install-config-about-ocr]]
+== About OpenShift Container Registry
 
-{product-title} is a fully-featured platform-as-a-service (PaaS) enterprise solution
-that includes an embedded container registry. However, it can alternatively be
-installed as a stand-alone container registry to run on-premise or in the cloud.
+{product-title} is a fully-featured platform-as-a-service (PaaS) enterprise
+solution that includes an integrated container registry called
+xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[OpenShift Container Registry] (OCR). Alternatively, instead of deploying {product-title}
+as a full PaaS environment for developers, you can install OCR as a stand-alone
+container registry to run on-premise or in the cloud.
 
-The stand-alone registry from {product-title} provides the following
-capabilities:
+When installing a stand-alone deployment of OCR, a cluster of masters and nodes
+is still deployed just as with a typical {product-title} installation. This
```

Comment:
My head got a little stuck with "deployed just as with a typical...". Can I suggest "still deployed as a typical..." instead? I think it might be the "with" that's confusing me.